scripts/create-database.js

```diff
@@ -0,0 +1,57 @@
+// scripts/create-database.js
```

Comment:
All the comments in this file are not really providing any value, they are just describing _how_ you are doing what you are and not _why_, which is a far more valuable thing to provide to whoever is trying to understand and work on your code. As it is, this is clear enough to understand without having to add the comments and so they could all be removed.

It looks to me like the whole file has been copy pasted from the track materials. Even if you do not understand fully what is going on here, blindly copy pasting is not a good thing as you could be adding bad or even malicious code, and creating side effects by accident that you may not even be aware of at first.

Its much more beneficial to you while learning these things to get into the idea of typing your work out as it helps you get accustomed to flow of work, and also is more engaging mentally